src/main/java/commands/ListCommand.java

```diff
@@ -0,0 +1,33 @@
+package commands;
+
+import skibidi.Command;
+import skibidi.Ui;
+import storage.Task;
+import storage.TaskStorage;
+
+/**
+ * Represents a command to list all tasks.
+ */
+public class ListCommand extends Command {
+    /**
+     * Executes the command to list all tasks.
+     *
+     * @param ui The user interface to interact with the user.
+     * @param storage The task storage to store the task.
+     * @return True to continue running the program.
+     */
+    @Override
+    public boolean execute(Ui ui, TaskStorage storage) {
+        StringBuilder list = new StringBuilder();
```

Comment:
Could the variable list be a little more descriptive as well? It might help with the readability of your code